apps/process/views.py

```diff
@@ -1,5 +1,123 @@
 from django.shortcuts import render
+from apps.calc.read_data import read
+from django.contrib.auth.decorators import login_required
+import json
+from .json import NumPyArangeEncoder
+
 
 # Create your views here.
-def index(request):
-    return render(request, "process/index.html")
\ No newline at end of file
+@login_required
+def fromDashboard(request):
+    if request.method == 'POST':
+        # Variablen aus dem Post-Request auslesen
+        jsonHeader = request.POST.get("jsonHeader", "")
+        jsonEinheiten = request.POST.get("jsonEinheiten", "")
+        zeitreihenSpalte = request.POST.get("zeitreihenSpalte", "")
+        jsonData = request.POST.get("jsonData", "")
+        saveExperiment = request.POST.get("saveExperiment", "")
+        datensatzName = request.POST.get("datensatzName", "")
+        erfassungsDatum = request.POST.get("erfassungsDatum", "")
+
+        # Das Experiment in das Objekt "measurement" einlesen
+        measurement = read.Measurement(jsonData,jsonHeader,jsonEinheiten,zeitreihenSpalte)
+
+        #measurement in Session-Variable speichern
+        request.session['measurementData'] = json.dumps(measurement.data, cls=NumPyArangeEncoder)
+        request.session['measurementHeader'] = json.dumps(measurement.colNames, cls=NumPyArangeEncoder)
+        request.session['measurementUnits'] = json.dumps(measurement.colUnits, cls=NumPyArangeEncoder)
+        request.session['measurementTimeIndex'] = json.dumps(measurement.timeIndex, cls=NumPyArangeEncoder)
+
+        #measurement.resample_data()
+        #measurement.gaussian_filter(1)
+        #measurement.butterworth_filter(1)
+        #measurement.fourier_transform(1)
+
+        # Daten zur Übergabe vorbereiten
+        dataForRender = {
+            'jsonHeader': jsonHeader,
+            'jsonEinheiten': jsonEinheiten,
+            'zeitreihenSpalte': zeitreihenSpalte,
+            'jsonData': jsonData,
+            #'newData': json.dumps(measurement.data, cls=NumPyArangeEncoder),
+            #'measurementObject': measurement,
+            #'saveExperiment': saveExperiment,
+            'datensatzName': datensatzName,
+            'erfassungsDatum': erfassungsDatum,
+            'expertMode': False #@TODO: wirklich den Expert-Mode auslesen! mit request.user
+        }
+        return render(request, "process/index.html", dataForRender)
+    else:
+        return render(request, "dashboard/index.html")
+
+
+
+
+def analysis(request):
+
+    if request.method == 'POST':
```

Comment:
Große If und kleine else immer umdrehen!

if not request.method == 'POST'
  return --->
Anderes zeug